geomstats/geometry/discrete_curves.py

```diff
@@ -464,7 +465,7 @@ def __init__(
                     " equipped with a metric."
                 )
         self.ambient_manifold = ambient_manifold
-        self.l2_metric = L2CurvesMetric(ambient_manifold=ambient_manifold)
+        self.l2_curve_metric = L2CurvesMetric(ambient_manifold=ambient_manifold)
```

Comment:
Maybe rename to `l2_curves_metric` for coherence, or `L2CurvesMetric` to `L2CurveMetric`?

Reply:
Yes, done!